src/data/faq.js

```diff
@@ -254,37 +254,31 @@ export default [
         answer: (
           <>
             <p>
-              The idea is to calculate the probability of not getting Fusion
-              stones in a book — other calculations are just small variations of
-              this one. To do that, we lay down all the different “drawing
-              sequences”; 64 in a Mythic book (2 different rarities to the power
-              of 6 draws), 4096 is a Classic book (4 different rarities to the
-              power of 6 draws). For example, ‘EEELEL’ (Epic, Epic, Epic,
-              Legendary, Epic, Legendary) is a valid and likely sequence for a
-              Mythic book. So at the core, it is a function that takes a drawing
-              sequence and returns the probability of getting this sequence, and
-              no fusion stones out of a book.
+              The book calculator calculates the probability of not getting
+              Fusion Stones (FS) out of a book (other calculations are just
+              small variations of this one). To do that, we lay down all the
```

Comment:
I tried to avoid “we” on purpose (as it’s particularly loose here) and use “it” (the calculator) instead. If you feel strongly that it improves the explanation to use “we“, we can keep using it though.